website/docs/d/atracker_settings.html.markdown

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Do we datasource for ibm_atracker_settings because i don't seee in provider.go only I see a resource for ibm_atracker_settings ...then this doc file should n't exists

Reply:
**Deleted**